src/protocol/templates/pingmessage_.h.in

```diff
@@ -17,12 +17,31 @@ class {{subclass_info.name}}{{message_type}}_{{message}} : public {{class_info.n
 public:
     {{subclass_info.name}}{{message_type}}_{{message}}(const PingMessage& msg) : {{class_info.name}} { msg } {}
     {{subclass_info.name}}{{message_type}}_{{message}}(const uint8_t* buf, const uint16_t length) : {{class_info.name}} { buf, length } {}
-    {{subclass_info.name}}{{message_type}}_{{message}}()
-        : {{class_info.name}} { {{8 + total_payload + 2}} }
+    {{subclass_info.name}}{{message_type}}_{{message}}(
+{%- for payload in messages[message_type]['all_msgs'][message].payload %}
+{% if is_vector(payload.type) %}
+{% if payload.vector.sizetype -%}
+            {{get_type_string(payload.vector.sizetype)}} {{payload.name}}_length
+{%- endif %}
+{% endif %}
+{% endfor %})
+        : {{class_info.name}} { {{8 + total_payload + 2}}
```

Comment:
`8 + total_payload + 2` Is creating a float and not a int.
```
//Old
        : PingMessage { 16 }
```
```
//New
        : PingMessage { 16.0 }
```

Reply:
interesting, I don't get it here.
And I get some errors with running python3.
Trying to fix it...